app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from typing import Union
+
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> Union[str, None]:
```

Comment:
Code Style: It's good practice to use type annotations for the function parameters and return types. In this case, the type of `friends` is not specified. It would be better to specify the expected type of elements in the list. If `friends` is expected to be a list of strings, use `List[str]`. If it is a list of some custom class instances, use `List[YourClass]`. It will improve the readability of your code and help to avoid potential type errors.